core/src/main/java/tech/pegasys/web3signer/core/service/http/metrics/OpenApiOperationsMetrics.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.core.service.http.metrics;
+
+import tech.pegasys.web3signer.core.metrics.Web3SignerMetricCategory;
+import tech.pegasys.web3signer.core.signing.KeyType;
+
+import org.hyperledger.besu.plugin.services.MetricsSystem;
+import org.hyperledger.besu.plugin.services.metrics.Counter;
+import org.hyperledger.besu.plugin.services.metrics.OperationTimer;
+
+public class OpenApiOperationsMetrics {
+  private static Counter signersLoadedCounter;
+
+  private final Counter malformedRequestCounter;
+  private final OperationTimer signingTimer;
+  private final Counter missingSignerCounter;
+
+  public OpenApiOperationsMetrics(final MetricsSystem metricsSystem, final KeyType keyType) {
```

Comment:
are these actually "OpenApiOperationsMetrics" - or maybe "RouteMetrics"? Not sure ... or are they really SignOperationMetrics?

Reply:
I think all of them are Sign operation metrics except for 1 which you had categorised under "http". Suggestions for class name??